systemtest/TESTING.md

```diff
@@ -0,0 +1,211 @@
+# Testing Strimzi
+
+This document gives a detailed breakdown of the testing processes and testing options for Strimzi within system tests. For more information about build process see [Hacking document](https://github.com/strimzi/strimzi-kafka-operator/blob/master/HACKING.md).
+
+<!-- TOC depthFrom:2 -->
+
+- [Pre-requisites](#pre-requisites)
+- [Package Structure](#package-structure)
+- [Test Phases](#test-phases)
+- [Available Test Groups](#available-test-groups)
+- [Environment Variables](#environment-variables)
+- [Use Remote Cluster](#use-remote-cluster)
+- [Helper Script](#helper-script)
+- [Running single test class](#running-single-test-class)
+- [Skip Teardown](#skip-teardown)
+
+<!-- /TOC -->
+
+## Pre-requisites 
+
+To run any system tests you need a Kubernetes or Openshift cluster available in your active kubernetes context. You can use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), [minishift](https://www.okd.io/minishift/), [oc cluster up](https://github.com/openshift/origin) or [CodeReady Contaners](https://github.com/code-ready/crc) 
```

Comment:
Line per sentence will make it easier to review and comment not just now but also in the next PRs ;-)

Reply:
done